blog/2024.9.9-QF.md

```diff
@@ -9,6 +9,20 @@ tags: [公告]
 
 <!-- truncate -->
 
+:::info 竞猜
+
+随着 Qualifier 的到来，OFFC24 Cirno 的竞猜也已启动！所有人均可参加，竞猜正确数目第一者可获得**一个月的 osu!supporter**！（若有并列，则选取投稿时间前三名）
+
+欢迎各位在 <Highlight color="#198dff">**[这里](https://challonge.com/zh_CN/tournaments/15088528/predictions/new)**</Highlight> 大胆填写对于赛事结果的预测与想象！
+
+您无需登录，只需要按网页的指示，将队伍拖入每轮比赛的预测结果空位中，然后在下方选择预测获胜方即可。
```

Comment:
Maybe we have to add more information on the format of the pick'em title. Users must include their osu! ID in the title.

Reply:
Below this line already has something about that.
Should I clarify it with more detail, or move them here?

Reply:
Resolved.